cmd/build.go

```diff
@@ -317,6 +317,9 @@ func buildCommandLoaderFilter(bundleMode bool, ignore []string) func(string, os.
 			if !info.IsDir() && strings.HasSuffix(abspath, ".tar.gz") {
```

Comment:
I can't comment on that line, but expand above to see line 316 where the non-bundle condition is checked.